### PR TITLE
tests updated to reflect root_enabled_history move to new mgmt api & client

### PR DESCRIPTION
**Pull request by ed-**
root history is no longer an appendage attached to instance details
furthermore it's just root_history and not root_enabled_history in the API

### CODE REVIEW

**Comment by cp16net** [comment]
+1

File: tests/integration/tests/api/instances_actions.py [hunk 1]
```diff
@@ -325,7 +327,7 @@ def test_successful_restart(self):
 @test(groups=[tests.INSTANCES, INSTANCE_GROUP, GROUP,
               GROUP + ".resize.instance"],
       depends_on_groups=[GROUP_START], depends_on=[create_user],
-      runs_after=[RebootTests])
+      runs_after=[RebootTests], enabled=False)
```
Comment (ed-) [inline tests/integration/tests/api/instances_actions.py]: whoops

**Comment by rnirmal** [comment]
+1

File: scripts/redstack [hunk 2]
```diff
@@ -753,6 +753,13 @@ function cmd_rd_client() {
     reddwarf-cli $@
 }
 
+function cmd_rd_mgmt_client() {
```
Comment (hub-cap) [inline scripts/redstack:756]: nice!

File: tests/integration/tests/api/instances_actions.py [hunk 3]
```diff
@@ -130,10 +130,12 @@ def create_user(self):
         users = [{"name": MYSQL_USERNAME, "password": MYSQL_PASSWORD,
                   "database": MYSQL_USERNAME}]
         self.dbaas.users.create(instance_info.id, users)
+
         def has_user():
             users = self.dbaas.users.list(instance_info.id)
             return any([user.name == MYSQL_USERNAME for user in users])
-        poll_until(has_user, time_out = 30)
+
+        poll_until(has_user, time_out=30)
```
Comment (hub-cap) [inline tests/integration/tests/api/instances_actions.py:138]: the pep gods would be proud

File: apidocs/src/resources/samples/db-mgmt-get-instance-details-response.json [hunk 4]
```diff
@@ -63,8 +63,6 @@ Date: Wed, 25 Jan 2012 21:58:25 GMT
         "hostname": "e09ad9a3f73309469cf1f43d11e79549caf9acf2.rackspaceclouddb.com", 
         "id": "692d8418-7a8f-47f1-8060-59846c6e024f", 
         "name": "xml_rack_instance", 
-        "root_enabled_at": "2012-01-25 21:58:11", 
-        "root_enabled_by": "examples", 
         "server_state_description": "active", 
         "status": "ACTIVE", 
         "updated": "2012-01-25T21:55:30Z",
```
Comment (TimSimpsonR) [inline apidocs/src/resources/samples/db-mgmt-get-instance-details-response.json:68]: I believe this is a first in RDLI. Kudos to you for updating these good sir.

**Pull request merged**